src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -755,7 +760,7 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
   }, [usingArchivedReport, usingCachedReport, report, isLoading, errorMessage]);
 
   return (
-    <Card id="automated-analysis-card" isRounded isCompact isExpanded={isCardExpanded}>
+    <Card id="automated-analysis-card" isRounded {...props} isExpanded={isCardExpanded}>
       <CardHeader
         isToggleRightAligned
```

Comment:
Since now we have action menu on the right, do you think we should move the expanded arrow to the left?

Something like this: https://www.patternfly.org/v4/components/card#expandable-cards

Reply:
We might need to adjust the card title tho since it shifts everything to left now.

How about this?

![Screenshot from 2022-12-08 17-52-11](https://user-images.githubusercontent.com/68053619/206583940-2a1dba96-5a54-4430-a587-2d56342437da.png)

Reply:
```diff
diff --git a/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx b/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx
index 7591686..14aa2fb 100644
--- a/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx
+++ b/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx
@@ -76,6 +76,7 @@ import {
   Checkbox,
   Grid,
   GridItem,
+  Label,
   LabelGroup,
   Split,
   SplitItem,
@@ -90,7 +91,7 @@ import {
   ToolbarItem,
   Tooltip,
 } from '@patternfly/react-core';
-import { OutlinedQuestionCircleIcon, Spinner2Icon, TrashIcon } from '@patternfly/react-icons';
+import { InfoCircleIcon, OutlinedQuestionCircleIcon, Spinner2Icon, TrashIcon } from '@patternfly/react-icons';
 import * as React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { filter, first, map, tap } from 'rxjs';
@@ -745,24 +746,17 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
       return undefined;
     }
     return (
-      <div
-        style={{
-          color: 'var(--pf-global--Color--200)',
-          fontWeight: 'lighter',
-          fontSize: '0.90em',
-          fontFamily: 'inherit',
-        }}
-      >
-        {`${usingArchivedReport ? 'Archived' : usingCachedReport ? 'Cached' : 'Active'} report name:`}{' '}
-        <span style={{ color: 'var(--pf-global--Color--100)', fontWeight: 'bold', fontStyle: 'italic' }}>{report}</span>
-      </div>
+      <>
+        <Label icon={<InfoCircleIcon />} color={'cyan'}>
+          {`${usingArchivedReport ? 'Archived' : usingCachedReport ? 'Cached' : 'Active'} report name=${report}`}
+        </Label>
+      </>
     );
   }, [usingArchivedReport, usingCachedReport, report, isLoading, errorMessage]);
 
   return (
     <Card id="automated-analysis-card" isRounded {...props} isExpanded={isCardExpanded}>
       <CardHeader
-        isToggleRightAligned
         onExpand={onCardExpand}
         toggleButtonProps={{
           id: 'automated-analysis-toggle-details',
@@ -771,13 +765,13 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
           'aria-expanded': isCardExpanded,
         }}
       >
-        <CardActions>{...props.actions || []}</CardActions>
-        <CardTitle component="h4">
-          <Split>
-            <SplitItem isFilled>Automated Analysis</SplitItem>
+        <CardTitle>
+          <Split hasGutter>
+            <SplitItem>Automated Analysis</SplitItem>
             <SplitItem>{reportSource}</SplitItem>
           </Split>
         </CardTitle>
+        <CardActions>{...props.actions || []}</CardActions>
       </CardHeader>
       <CardExpandableContent>
         <Stack hasGutter>
```

Reply:
On this note, do you think we should move the caret of `TargetSelect` to the left too?